_guide/_pages/accessibility-scanning.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: Accessibility Scanning
+layout: post
```

Comment:
I suggest setting these as [front matter defaults](https://jekyllrb.com/docs/configuration/front-matter-defaults/).

Reply:
Perfect – thanks! I applied to the upstream template and have removed all those new layout directives.